src/main/resources/openapi/transcriptions.yaml

```diff
@@ -829,6 +829,43 @@ paths:
                 type: "USER_MANAGEMENT_108"
                 title: "The provided transcription does not exist"
                 status: 404
+
+  /admin/transcription-documents/{transcription_document_id}:
+    get:
+      tags:
+        - Transcription
+      summary: Gets a transcription document id
+      operationId: getByDocumentId
+      parameters:
+        - name: transcription_document_id
+          in: path
+          required: true
+          schema:
+            type: integer
+      responses:
+        '200':
+          description: OK
+          content:
+            application/json:
+              schema:
+                  $ref: '#/components/schemas/SearchTranscriptionDocumentByIdResponse'
```

Comment:
should the response be called "GetTranscriptionDocumentByIdResponse" to match the endpoint?